modules/web/src/app/node-data/basic/provider/kubevirt/topology-spread-contraint-form/component.ts

```diff
@@ -115,10 +125,33 @@ export class TopologySpreadConstraintFormComponent extends BaseFormValidator imp
   }
 
   deleteConstraint(index: number): void {
+    this.removedConstraints?.push(this.constraints[index]);
+
     this.constraintArray.removeAt(index);
     this._updateConstraints();
   }
 
+  isConstraintsChanged(index: number): boolean {
+    const [maxSkew, topologyKey, whenUnsatisfiable] = [
+      this.constraints[index]?.maxSkew,
+      this.constraints[index]?.topologyKey,
+      this.constraints[index]?.whenUnsatisfiable,
+    ];
+    const [initialMaxSkew, initialTopologyKey, initialWhenUnsatisfiable] = [
+      this.initialConstraint[index]?.maxSkew,
+      this.initialConstraint[index]?.topologyKey,
+      this.initialConstraint[index]?.whenUnsatisfiable,
+    ];
+    if (maxSkew && topologyKey && whenUnsatisfiable) {
+      return (
+        maxSkew === initialMaxSkew &&
+        (topologyKey !== initialTopologyKey || whenUnsatisfiable !== initialWhenUnsatisfiable) &&
+        this._dialogModeService.isEditDialog
```

Comment:
so the max skew is a km-number-stepper already have it's own value changed directive so no need to add indicator when it's changed but i already fix that so it can be more clear